Doppler.PushContact/appsettings.json

```
@@ -17,8 +17,7 @@
     "PublicKeysFilenameRegex": "\\.xml$"
   },
   "PushMongoContextSettings": {
-    "Host": "REPLACE_WITH_MONGODB_HOST",
-    "Username": "REPLACE_WITH_MONGODB_USERNAME",
+    "ConnectionString": "mongodb+srv://USERNAME:PASSWORD@HOST",
```

Ya que siempre se va a reemplazar por `Password`, si bien es ficticio, lo eliminaría para evitar confución. Qué opinás @Manu0912 ?
```suggestion
    "ConnectionString": "mongodb+srv://USERNAME@HOST",
```

---

para lo que es qa int y prod me parece que si, pero en el caso de que alguien haga pruebas locales no puede llegar a confundir mas si lo saco?

---

Si alguien hace pruebas locales, este `PASSWORD` tampoco sería utilizado, no?

---

Pensandolo bien, creo que si, es mejor sacarlo. Ahora meto cambio